Common/Data/MobRegistry.cs

```diff
@@ -21,6 +21,51 @@ public void Load(Mod mod)
 			Console.WriteLine($"Mob With Key: {entry.Key} - Registered");
 		}
 	}
+
+	/// <summary>
+	/// Provides mod NPC data for the Mob Registry.
+    ///
+    /// Note: Must be called within a PostSetupContent() method belonging to a Mod or ModSystem class.
+	/// </summary>
+    public static void PostLoad()
```

Comment:
The way this is set up looks like it's meant to be reused for other mods or something, and the comment above also seems like that's the case. Would it be an issue to have a mod and path parameter just to benefit this use case? 
This'd allow other mods to register mob data easily with `MobRegistry.PostLoad(myModInstance, "Path/To/Data");` if they so wish to.